src/engine/source/cmds/src/start.cpp

```diff
@@ -198,19 +200,24 @@ void runStart(ConfHandler confManager)
 
         // KVDB
         {
-            kvdb = std::make_shared<kvdb_manager::KVDBManager>(kvdbPath, metrics);
+            kvdbManager::KVDBManagerOptions kvdbOptions { "/var/ossec/etc/kvdb2/", "kvdb" };
```

Comment:
Please, let's try to avoid using hardcoded values, I think it is better to declare an expression, at the beginning of the file or in a configuration file such as `defaultSettings.hpp`, that can be easily modified

Reply:
The same goes for all the occurrences